### DISCUSSION TITLE
Creating an Automated Installation Script for Apktool on Linux

### DISCUSSION THREAD

**Discussion by Gabriel712** [iBotPeaches/Apktool#3590] [Ideas]
Installing Apktool manually can be a real hassle since you have to download two separate files and manually adjust permissions. With an automated installation script, you get rid of that headache by turning multiple steps into one quick and simple action. It's a significant save on time and effort!

**Comment by iBotPeaches** [comment]
I've tried to avoid that as a lot of professionals in this field - do not want anything automated installing stuff anywhere. They prefer to control (or ignore) the helper scripts and use the jar how they wish.
